[docs] Added a warning about changing ClusterConfiguration (#9770)
deckhouse/deckhouse: docs/documentation/pages/installing/CONFIGURATION.md

Reference of the resources used during [Deckhouse installation](./).

{% alert level="danger" %}

> Do not change the parameters `internalNetworkCIDRs`, `serviceSubnetCIDR`, `podSubnetNodeCIDRPrefix`, `podSubnetCIDR` in a running cluster. Deploy a new cluster, if you have to change these parameters.

Reviewer: ```suggestion
Do not change the `internalNetworkCIDRs`, `serviceSubnetCIDR`, `podSubnetNodeCIDRPrefix`, `podSubnetCIDR` parameters in a running cluster. If you do need to change them, deploy a new cluster.
```

> We do not recommend changing the parameters `internalNetworkCIDRs`, `serviceSubnetCIDR`, `podSubnetNodeCIDRPrefix`, `podSubnetCIDR` in a running cluster. It is recommended to create a new cluster to change these parameters.

Reviewer 2: ```suggestion
Do not change the parameters `internalNetworkCIDRs`, `serviceSubnetCIDR`, `podSubnetNodeCIDRPrefix`, `podSubnetCIDR` in a running cluster. Deploy a new cluster, if you have to change these parameters.
```
